source/uri-options/uri-options.rst

```diff
@@ -71,6 +71,14 @@ is specified with an SRV URI, because the URI may resolve to multiple
 hosts. The driver MUST allow specifying ``directConnection=false`` URI
 option with an SRV URI.
 
+Non-SRV URI with srvServiceName URI option
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The driver MUST report an error if the ``srvServiceName`` URI option is
+specified with a non-SRV URI (i.e. ``mongodb://``) because SRV lookup
+only occurs with SRV URIs. The driver MUST allow specifying the ``srvServiceName``
+URI option with an SRV URI.
```

Comment:
I was going to ask if we needed to explicitly prohibit `srvServiceName` in TXT records, but I was reminded that there is already an [explicit list of supported URI options](https://github.com/mongodb/specifications/blob/master/source/initial-dns-seedlist-discovery/initial-dns-seedlist-discovery.rst#default-connection-string-options) -- so nothing to worry about there.